plugin/minimal/README.md

```diff
@@ -0,0 +1,74 @@
+# minimal
+
+## Name
+
+*minimal* - minimizes the size of the DNS query response 
+
+## Description
+
+The minimal plugin controls the addition of records to the AUTHORITY and ADDITIONAL section of the dns query response.
```

Comment:
```suggestion
The *minimal* plugin tries the minimize the size of the response. Depending the response type it removes resource records from the  AUTHORITY and ADDITIONAL sections.
```

Reply:
@miekg updated suggestion:
```suggestion
The *minimal* plugin tries to minimize the size of the response. Depending the response type it removes resource records from the AUTHORITY and ADDITIONAL sections.
```
There were two spaces before AUTHORITY in your suggestion @miekg and I changes `tries the minimize` -> `tries to minimize` :)